CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Update `k8sclient` dependency to `3.1.0`.
 - Update `tenantcluster` dependency to `2.0.0`.
+- Update Kubernetes dependencies to v1.18.5.
```

Comment:
Changelog is wrong in master. Can you remove the 2 lines above? As they are already released.

Reply:
thanks for pointing out, I will delete that.